docs/rtd/Makefile

```diff
@@ -12,7 +12,110 @@ BUILDDIR      = build
 help:
 	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
-.PHONY: help Makefile
+clean:
+	rm -fr build/
+	rm -f source/man/podman-*.1.md
+
+copy:
+	cp -v ../podman-attach.1.md source/man/
+	cp -v ../podman-build.1.md source/man/
+	cp -v ../podman-commit.1.md source/man/
+	# container
+	cp -v ../podman-container-checkpoint.1.md  source/man/
+	cp -v ../podman-container-exists.1.md source/man/
+	cp -v ../podman-container-restore.1.md source/man/
+	cp -v ../podman-container-cleanup.1.md source/man/
+	cp -v ../podman-container-prune.1.md source/man/
```

Comment:
Where will `source/man/` land? Will this be on the installed machine and/or here somewhere in GitHub?  Will it be under the docs directory?

Reply:
it is only for the build that readthedocs does

Reply:
Is there a reason for copying these?  
Why isn't it just cp -v ../podman*.md source/man
This will be a pain for anyone adding a new man page.